flang/lib/Optimizer/CodeGen/CodeGen.cpp

```diff
@@ -1635,6 +1635,24 @@ struct ValueOpCommon {
     }
   }
 
+  static llvm::SmallVector<mlir::Attribute>
+  collectIndices(mlir::ConversionPatternRewriter &rewriter,
```

Comment:
Could const `rewriter`.

Reply:
It looks like it was used on line 1650.

Reply:
The rewriter is passed around as a non-const generally, so I think it might be better to be consistent (and avoid any const correctness warnings).